PerformanceStatus.fsh

```diff
@@ -16,19 +16,21 @@ Description:    "The Karnofsky Performance Status (KPS) is a tool used to measur
 * basedOn only Reference (ServiceRequest | CarePlan)
 * partOf only Reference (Procedure)
 * category from http://hl7.org/fhir/ValueSet/observation-category (extensible)
-* code = LNC#89245-5 "Karnofsky Performance Status [Interpretation]"
+* code = LNC#89243-0 "Karnofsky Performance Status score"
 * subject only Reference(USCorePatient)
 * effective[x] only dateTime or Period
 * performer only Reference(Practitioner)
-* value[x] only CodeableConcept
-* valueCodeableConcept from KarnofskyPerformanceStatusVS (required)
+* value[x] only integer
+* interpretation from http://loinc.org/vs/LL4986-7 (extensible)
+
+// * valueCodeableConcept from KarnofskyPerformanceStatusVS (required)
```

Comment:
You must remove KarnofskyPerformanceStatusVS from the value set file as well, or it will be included in the IG.

Reply:
good catch. lemme do that.